docker/Dockerfile

```diff
@@ -22,9 +22,6 @@ WORKDIR ./trezor-user-env
 RUN ls /
 RUN ls /trezor-user-env
 
-# updates the nix-channel to have latest changes
-RUN nix-channel --update
```

Comment:
i suppose we could keep this in case we run in to other version discrepancies when running other stuff for example. But i'll leave it up to you.

Reply:
I guess you are right, it doesn't hurt. The bad thing is we won't know now whether the pinning works or not, but let's find out later :). I put it back.